### ISSUE TITLE
Behaviour of missing values in data

### ISSUE THREAD

**Issue by rhiaro** [openownership/data-standard#559]
Currently the schema excludes the possibility of optional fields being present with null values. It seems like the expected behaviour is that where data is missing, the fields must always be excluded.

I just want to check this is intentional.

(I could imagine, for example, a system which outputs data where all fields are present, and the missing ones are simply left blank. At the moment that wouldn't validate - but there's an easy change to the schema to ensure it would.)

Either way, if this is intended behaviour, we should make sure it's documented, so that the situation I just imagined never comes to pass!

**Comment by kathryn-ods** [comment]
I agree optional fields should be allowed to be blank. I don't know whether this was an intentional choice previously.
